dynamiqs/solvers/solver.py

```diff
@@ -128,6 +101,16 @@ def _save_exp_ops(self, y: Tensor):
         if len(self.exp_ops) > 0:
             self.exp_save[..., self.save_counter] = bexpect(self.exp_ops, y)
 
+    def clear_cache(self):
```

Comment:
This is quite a dangerous way to do what you want, let's keep this for now, I'll propose something cleaner soon.